src/Evolution/Systems/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -43,5 +43,6 @@ target_link_libraries(
   Domain
   )
 
+add_subdirectory(BoundaryCorrections)
```

Comment:
While you're here, can you fix the alphabetization of the directories? (=> swap the two below)